packages/playground/src/weblets/profile_manager.vue

```diff
@@ -284,13 +287,93 @@ defineProps({
 });
 defineEmits<{ (event: "update:modelValue", value: boolean): void }>();
 
+let mountedTimeout: any;
+watch(
+  () => props.modelValue,
+  m => {
+    if (m) {
+      if (mountedTimeout) {
+        clearTimeout(mountedTimeout);
+      }
+      mountedTimeout = setTimeout(() => {
+        mounted();
+      });
+    }
+  },
+);
+
+function mounted() {
+  if (isStoredCredentials()) {
+    activeTab.value = 0;
+    const credentials: Credentials = getCredentials();
+    const sessionPassword = sessionStorage.getItem("password");
+
+    if (!sessionPassword) return;
+
+    password.value = sessionPassword;
+
+    if (credentials.passwordHash) {
+      return login();
+    }
+  } else {
+    activeTab.value = 1;
+    return;
+  }
+}
+
+function onKeyDown() {
+  if (password.value.length >= 6) {
+    if (activeTab.value === 0) [login()];
```

Comment:
Why do we wrap login call with array [login()]?

Reply:
Updated